packages/navitem/src/react/__specs__/index.spec.tsx

```diff
@@ -28,26 +26,11 @@ describe('NavItem', () => {
     const ref = React.createRef<HTMLButtonElement>()
 
     render(
-      <NavItem renderContainer={props => <button ref={ref} {...props} />}>
+      <NavItem renderContent={props => <button ref={ref} {...props} />}>
```

Comment:
Almost seems like this test case can go away.  Now it's reading as "`<button />` can render with refs", which must, of course be true.

Well, I guess if we didn't use renderContent() or messed with it somehow, _maybe_ that's less-obviously true.

Reply:
yeah makes sense. let me noodle on that a bit. i'll probably remove the test